FFOSBR.js

```diff
@@ -12,67 +12,78 @@
 
 },{"./settings":2}],2:[function(require,module,exports){
 function Settings() {
-  var timeInMilliSec = 0;
 
   var options = {
-    photos: false,
-    videos: false,
-    contacts: false,
-    text: false,
-    intervalTime: 0, // pass in value in hours
+    photos: true,
+    videos: true,
+    contacts: true,
+    text: true,
+    intervalTime: 24, // pass in value in hours
     id: 0,
     registeredTimer: false,
-    repeat: false
+    repeat: true
   };
 
+  // Load options if present
+  (function() {
+    var retrievedOptions = localStorage.getItem('ffosbrOptions');
 
+    if (retrievedOptions !== null) {
+      options = JSON.parse(retrievedOptions);
+    }
+  })();
 
-  Settings.prototype.Options = function(myDictionary) {
+  Settings.prototype.options = function(myDictionary) {
     if (!myDictionary) {
       return options;
     }
 
-
-    if (myDictionary.photos !== undefined && typeof myDictionary.photos === "boolean") {
+    if (myDictionary.photos !== undefined &&
```

Comment:
All the if-statements can be reduced to "if (typeof myDictionary.photos === 'boolean') ..." because "undefined" is also a type returned by typeof. Meaning, there's no harm in calling typeof on an undefined variable and if its type is "boolean" then you know it isn't undefined anyway.